cabal.project

```diff
@@ -319,92 +319,92 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 670c734f0aa74a3feab8ff8dd6ee8793b2e29e1c
-  --sha256: 0q2c07h8ld3jhs92c0pzqlmgmlbyccnwkjs7d6v6vsarlz9hizp0
+  tag: 5a42504bc412d846f5b214aff1bf12ec3c542e5c
+  --sha256: 11fwf2k1bi668a17ngxr6f8csvm2yhjmrcn3rgv14bndc2jl5vwl
```

Comment:
This refers to a [draft PR](https://github.com/input-output-hk/ouroboros-network/pull/2449/) on `ouroboros-network`, so this still has to be updated.